aim/artifacts/model.py

```diff
@@ -23,6 +25,11 @@ def load(path: str) -> (bool, Any):
             model_path = os.path.join(working_dir, path)
             if not os.path.isfile(model_path):
                 return False, None
+        
+        # Create temporary directory
+        dir_name = 'tmp_model'
```

Comment:
It is better to use `NamedTemporaryFile` or `TemporaryDirectory` with tempfile package

Reply:
Got it! Changed to use TemporaryDirectory instead.